### PR TITLE
next/452/70x/20240524/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #11025

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1854

### CODE REVIEW

**Review by jufajardini — APPROVED** [review]
Number and list of commits seem to match;
Number of changes lines matches.

File: src/util-port-interval-tree.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * \file
+ *
+ * \author Shivani Bhardwaj <shivani@oisf.net>
+ */
+
+#ifndef __UTIL_INTERVAL_TREE_H__
+#define __UTIL_INTERVAL_TREE_H__
+
+#include "detect-engine-port.h"
+
```
Comment (jasonish) [inline src/util-port-interval-tree.h:28]: Something to fix in master? An empty header file?

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPR1_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPR1_stats_chk | | |
| .uptime | 649 | 688 | 106.01% | 

Pipeline 20783

File: src/util-port-interval-tree.h (hunk 1 above)
Comment (inashivb) [reply src/util-port-interval-tree.h:28]: Done in https://github.com/OISF/suricata/pull/11143

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 20789

**Pull request merged**